.devcontainer/devcontainer.json

```
@@ -1,32 +1,35 @@
 // For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
-// https://github.com/microsoft/vscode-dev-containers/tree/v0.122.1/containers/python-3
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/python-3
 {
     "name": "Python 3",
     "build": {
         "dockerfile": "Dockerfile",
         "context": "..",
-        "args": { "VARIANT": "3" }
+        "args": {
+            "VARIANT": "3"
```

I believe we do not need Node?

```suggestion
            "VARIANT": "3"
            "NODE_VERSION": "none"
```

---

Now I added this option too.